### PR TITLE
Remove DAFFODIL_HOME since IDEs don't need it

### PR DESCRIPTION
**Pull request by olabusayoT**
- use "." for daffodilRoot instead

DAFFODIL-2836

### CODE REVIEW

File: daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/Util.scala [hunk 1]
```diff
@@ -52,7 +52,7 @@ object Util {
 
   private val isWindows = System.getProperty("os.name").toLowerCase().startsWith("windows")
 
-  private val daffodilRoot = sys.env.getOrElse("DAFFODIL_HOME", ".")
+  private val daffodilRoot = "."
```
Comment (stevedlawrence) [inline daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/Util.scala:55]: Can we delete daffodilRoot and it's uses? Looks like its only used in a couple of places, where a value of "." is essentially a no-op.

**Review by stevedlawrence — APPROVED** [review]
+1

File: daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/Util.scala (hunk 1 above)
Comment (olabusayoT) [reply daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/Util.scala]: Done

File: daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/Util.scala [hunk 2]
```diff
@@ -67,7 +65,7 @@ object Util {
    * operating system
    */
   def path(string: String): Path = {
-    Paths.get(daffodilRoot, string)
+    Paths.get(string)
```
Comment (stevedlawrence) [inline daffodil-cli/src/test/scala/org/apache/daffodil/cli/cliTest/Util.scala:68]: Can you remove daffodilRoot from the comment too? Also, while you're in this section can you replace "line separator" with "path separator" in this comment, I think "line" was probably a typo?

**Review by mbeckerle — APPROVED** [review]
+1

**Pull request merged**